rust/tests/behaviour/steps/lib.rs

```diff
@@ -228,8 +223,8 @@ impl Context {
                 .await
                 .unwrap()
                 .into_iter()
-                .filter(|user| user.username != Context::ADMIN_USERNAME)
-                .map(|user| self.driver.as_ref().unwrap().users().delete(user.username)),
+                .filter(|user| user.name != Context::ADMIN_USERNAME)
```

Comment:
I'd introduce `fn is_admin(&self)` for `User`.

Reply:
Given we're not sure how the authentication are going to be extended, I'd rather not introduce too many public-facing API that may be ended up being changed in the future.

Reply:
Fine, it's driver and it's tests. It's more like `is_root_admin` to me here, and should probably be a server call. I'm happy to leave it this way here